### PR TITLE
 new: Add support for Parent/Child account switching

### PR DESCRIPTION
**Pull request by jriddle-linode**

### CODE REVIEW

File: plugins/module_utils/doc_fragments/child_account_info.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"""Documentation fragments for the vpc_info module"""
```
Comment (yec-akamai) [inline plugins/module_utils/doc_fragments/child_account_info.py:1]: nit:
```suggestion
"""Documentation fragments for the child_account_info module"""
```

File: plugins/module_utils/doc_fragments/child_account_list.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+"""Documentation fragments for the vpc_list module"""
```
Comment (yec-akamai) [inline plugins/module_utils/doc_fragments/child_account_list.py:1]: nit:
```suggestion
"""Documentation fragments for the child_account_list module"""
```

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,8 @@
-linode-api4>=5.13.1
+# TODO: Revert before merging to dev
+# linode-api4>=5.13.1
+git+https://github.com/linode/linode_api4-python@proj/parent-child
```
Comment (yec-akamai) [inline requirements.txt:3]: We probably want to revert this before merging into dev.

**Comment by jriddle-linode** [comment]
Doc's Validation is complaining because linode-py hasn't been updated yet.

**Review by zliang-akamai — APPROVED** [review]

**Comment by lgarber-akamai** [comment]
@jriddle-linode The Python SDK release has been cut so this should be ready to bump 👍

**Review by lgarber-akamai — APPROVED** [review]
LGTM!

**Review by lgarber-akamai — APPROVED** [review]

**Pull request merged**